packages/orun/orun.0.0.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Run benchmarks and measure performance"
+maintainer: "Shakthi Kannan <author@shakthimaan.com>"
+authors: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+license: "MIT"
+homepage: "https://github.com/ocaml-bench/orun"
+bug-reports: "https://github.com/ocaml-bench/orun/issues"
+dev-repo: "git+https://github.com/shakthimaan/orun"
+depends: [
+  "ocaml" {>= "4.07"}
+  "dune" {>= "1.2"}
+  "conf-libdw" {build}
```

Comment:
```suggestion
  "conf-libdw" {build & os = "linux"}
```
Looking at the code detecting libdw, am i correct in assuming that libdw is only used on linux?

Reply:
I just tried and on macOS the package fails with:
```
dune build -p orun
      ocamlc .profiler.objs/byte/profiler.{cmi,cmo,cmt} (exit 2)
(cd _build/default && /usr/local/bin/ocamlc.opt -w -40 -w -49 -nopervasives -nostdlib -g -bin-annot -I .profiler.objs/byte -no-alias-deps -o .profiler.objs/byte/profiler.cmo -c -impl profiler.ml-gen)
File "profiler.ml-gen", line 1:
Error: Could not find the .cmi file for interface profiler.mli.
```
I'll add the constraint to make this package linux only

Reply:
Yes. It is for Linux only.